string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,55 @@
+/* exported titleCase */
+
+function titleCase(title) {
+  var titleWords = [
+    'and',
+    'or',
+    'nor',
+    'but',
+    'a',
+    'an',
+    'the',
+    'as',
+    'at',
+    'by',
+    'for',
+    'in',
+    'of',
+    'on',
+    'per',
+    'to'
+  ];
+  var newString = '';
+  var titleArray = title.split(' ');
+  for (var i = 0; i < titleArray.length; i++) {
+    if (
+      titleWords.includes(titleArray[i]) &&
+      newString[newString.length - 2] !== ':'
+    ) {
+      newString += titleArray[i] + ' ';
+      continue;
+    } else if (titleArray[i] === 'javascript') {
+      newString += 'JavaScript ';
+      continue;
+    } else if (
+      titleArray[i] === 'Javascript:' ||
+      titleArray[i] === 'javascript:'
+    ) {
+      newString += 'JavaScript: ';
+      continue;
```

Comment:
All of these situations seem like there is some overlap. They all seem to be checking for `JavaScript` and `:` in some form. Can this be streamlined in any way?